### ISSUE TITLE
convert an .obj file to .navmesh

### ISSUE THREAD

**Issue by UDKMan** [recastnavigation/recastnavigation#270]
Hi Guns！How to convert an .obj file to .navmesh. Please help me!

**Comment by hymerman** [comment]
Depends what you mean! Nothing in this project uses ".navmesh" as an extension so I'm not sure what you want to be in such a file. Are you using another tool that uses this as a file format?

**Comment by darkwere** [comment]
Let me make a guess! It seems to me, that UDKMan is talking about kbengine project (https://github.com/kbengine/kbengine). There is a need for a game server to contain a level geometry, in order for monsters, for example, to use a walkable areas and not to fall down through level or not to walk through the buildings.
KBengine uses recastnavigation 1.4 for navigation mesh handling.
There are at least two realizations of client part for kbengine server. One is for Unity3D, the other for Unreal Engine. The question is, in my opinion, about how to export the level geometry from U3D or UE4 into .navmesh, which is suitable for kbengine.

I was working on this problem for some time. I use Unreal Engine 4. At this time I have the following recipe to generate .navmesh in UE4.
1. Select Landscape and shrink transform scale 100 times, because UE4 uses cm for measure unit.
2. Replace Landscape at X/100, Y/100, Z/100 coordinates.
3. Select File -> Export Selected.
4. Select OBJ file format and save it somewhere on a HD.
5. I used RecastDemo prog to generate tile navmesh.
6. Rename all_tiles_navmesh.bin to your_file_name.navmesh and place it at kbengine res/spaces folder.
Note: If you do not shrink 100 times level geometry before export in UE4, your 500m x 500m level will become 50km x 50km landscape. RecastDemo will hang when you try to generate a navmesh.

At this moment I have only this solution to generate .navmesh, but I've stuck with one problem, which I need help with. I've recorded a video, illustrating this problem, and hope that somebody help me with advise on it! https://yadi.sk/i/MC4CIeit3KLKD4
